libraries/AP_NavEKF3/AP_NavEKF3_core.cpp

```diff
@@ -1980,3 +1990,14 @@ Vector3f NavEKF3_core::calcRotVecVariances()
 
     return rotVarVec;
 }
+
+void NavEKF3_core::bestRotationOrder(rotationOrder &order)
+{
+    if (fabsf(prevTnb[2][0]) < fabsf(prevTnb[2][1])) {
```

Comment:
I'm wondering what will happen when these two values are very close to each other, does noise in these values  introduce the possibility of 'flapping' back and forth between these two different code paths.? ie is it possible that jitter here would cause TAIT_BRYAN_321 on one iteration, then TAIT_BRYAN_312, the back to TAIT_BRYAN_321 on the next etc..     it kinda appears to me that it's most likely to occur when the board orientation is at around 45degrees from level or vertical..?

Reply:
It doesn't matter as long as it is consistent for the operations using it on that frame, ie if performing a yaw reset involving multiple steps, the same rotation must be used throughout.